### DISCUSSION TITLE
What is meant by Zero negative electrode porosity cut-off?

### DISCUSSION THREAD

**Discussion by wass-cloud** [pybamm-team/PyBaMM#3158] [Q&A]
I've been getting this error for a bit and I was curious why it occurs and how it can be avoided.

**Comment by brosaplanella** [comment]
It means that the porosity in the negative electrode gets to zero (i.e. pores get clogged) which triggers the simulation to stop. The model is not tested beyond this, so I don't know if removing the event to solve beyond pore clogging is only an issue of numerical stability (like with electrolyte depletion) or if there are more fundamental problems.

**Reply by wass-cloud** [reply]
I see, the issue lies within the capacity increasing parameters I set. Thank you!
